src/pages/accounting/BalancesTable.tsx

```diff
@@ -1,79 +1,90 @@
-                <TableCell style={{ textAlign: 'right' }}>Balance (BZZ)</TableCell>
-                <TableCell align="right"></TableCell>
-              </TableRow>
-            </TableHead>
-            <TableBody>
-              {props.peerBalances?.balances.map((peerBalance: PeerBalance) => (
-                <TableRow key={peerBalance.peer}>
-                  <TableCell>{peerBalance.peer}</TableCell>
-                  <TableCell
-                    style={{
-                      color: fromBZZbaseUnit(peerBalance.balance) > 0 ? '#32c48d' : '#c9201f',
-                      textAlign: 'right',
-                      fontFamily: 'monospace, monospace',
-                    }}
-                  >
-                    {fromBZZbaseUnit(peerBalance.balance).toFixed(7).toLocaleString()}
-                  </TableCell>
-                  <TableCell align="right"></TableCell>
-                </TableRow>
-              ))}
-            </TableBody>
-          </Table>
-        </TableContainer>
-      )}
-    </div>
+    <TableContainer component={Paper}>
+      <Table className={classes.table} size="small" aria-label="simple table">
+        <TableHead>
+          <TableRow>
+            <TableCell>Peer</TableCell>
+            <TableCell style={{ textAlign: 'right' }}>Outstanding Balance</TableCell>
+            <TableCell style={{ textAlign: 'right' }}>Settlements Sent / Received</TableCell>
+            <TableCell style={{ textAlign: 'right' }}>Total</TableCell>
+            <TableCell style={{ textAlign: 'right' }}>Uncashed Amount</TableCell>
+            <TableCell style={{ textAlign: 'right' }}></TableCell>
+          </TableRow>
+        </TableHead>
+        <TableBody>
+          {accounting.map(({ peer, balance, received, sent, uncashedAmount }) => (
+            <TableRow key={peer}>
+              <TableCell>
+                <div style={{ display: 'flex' }}>
+                  <small>
+                    <PeerDetailDrawer peerId={peer} />
+                  </small>
+                  <ClipboardCopy value={peer} />
+                </div>
+              </TableCell>
+              <TableCell className={classes.values}>
+                <span
+                  style={{
+                    color: balance > 0 ? '#32c48d' : '#c9201f',
+                  }}
+                >
+                  {fromBZZbaseUnit(balance).toFixed(7).toLocaleString()}
+                </span>{' '}
+                BZZ
+              </TableCell>
+              <TableCell className={classes.values}>
+                -{fromBZZbaseUnit(sent).toFixed(7)} / {fromBZZbaseUnit(received).toFixed(7)} BZZ
+              </TableCell>
+              <TableCell className={classes.values}>
+                <span
+                  style={{
+                    color: balance + received - sent > 0 ? '#32c48d' : '#c9201f',
+                  }}
+                >
+                  {fromBZZbaseUnit(balance + received - sent).toFixed(7)}
```

Comment:
Maybe the expression `balance + received - sent` could be extracted as a variable